lib/datadog.py

```diff
@@ -0,0 +1,64 @@
+from datadog_api_client.configuration import Configuration
+from datadog_api_client.api_client import ApiClient
+from datadog_api_client.v2.api import metrics_api
+from datadog_api_client.v2.model.metric_point import MetricPoint
+from datadog_api_client.v2.model.metric_series import MetricSeries
+from datadog_api_client.v2.model.metric_payload import MetricPayload
+from datadog_api_client.v2.model.metric_intake_type import MetricIntakeType
+import os, time
+
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
I'd recommend doing this in `main.py`, not here.

Generally, it's easier to work with code if global/external side-effects are kept to a limited set of files and libraries are kept purely focused on one task.

Reply:
Addressed in [`6ee75f1` (#13)](https://github.com/iFixit/lighthouse-docker/pull/13/commits/6ee75f1ab4680278807a72dde0bc2a067be55a62)